zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/SQLPPGeneratorTest.scala

```diff
@@ -695,4 +695,24 @@ class SQLPPGeneratorTest extends Specification {
         """.stripMargin.trim)
     }
   }
+
+  "SQLPPGenerator deleteRecord" should {
+    "generate the delete query without filter" in {
+      val sql = parser.generate(DeleteRecord(TwitterDataSet, Seq.empty), Map(TwitterDataSet -> TwitterDataStore.TwitterSchema))
+      removeEmptyLine(sql) must_== unifyNewLine(
+        """
+          |delete from twitter.ds_tweet t;
```

Comment:
Sure, but it should be fine. This query is not exposed to public, but for internal use only...